### ISSUE TITLE
docs: update log guide for 5.x

### ISSUE THREAD

**Issue by quisquous** [quisquous/cactbot#573]
There were a number of log changes in the 5.x version of the ffxiv plugin, in terms of object ids being included in the formatted log output for gains/loses effect and added combatant.

There's also a few new log lines that need to be added like 249 (settings) and 250 (current process information).

**Comment by JLGarber** [comment]
Does Ravahn have this documented anywhere, or is this a matter of hunting through for things? I'm happy to help with this as I can, but I don't know enough about the log format to "just know what needs fixing". I suppose I can start with searching in the XIV Plugin Discord server...

**Comment by quisquous** [comment]
No, this isn't documented anywhere.  You just have to kind of run stuff and figure it out based on reading log lines.  That's how the log guide got written in the first place.  <_<

Additional new log lines now include:
* 0xF9 (settings)
* 0xFA (process information)
* 0x0C (player stats)
* 0x24 (limit break)
* 0x25, 0x26, 0x27: new things added in 2.0.4.0 related to hp/mp/status??

Additionally: 
* 0x15 and 0x16 have heading and "sequence number" at the end of them.
* 0x14 has the caster id in it
* 0x1A and 0x1E have the caster object id at the front

**Comment by JLGarber** [comment]
Oh, so lots of hunting for stuff I don't know. Fair enough, I can do that. No idea on how long it will take me to get any of that sorted, but are you okay with pull requests that update only one log line? This way if anyone also wants to pick up at some point, they can be reasonably sure they aren't duplicating effort for more than one line type.

**Comment by quisquous** [comment]
Pull requests with one line are just fine.  No need to get everything done before updating documentation.

**Comment by ngld** [comment]
0x25 is `NetworkEffectResult`, 0x26 is `NetworkStatusList`, 0x27 is `NetworkUpdateHp`

Unless I counted wrong somewhere, 0x1A is `NetworkBuff`, 0x1E is `NetworkBuffRemove` and they have the following format:
`BuffID|buffName|Duration|ActorID|ActorName|TargetID|TargetName|BuffExtra|TargetMaxHP|ActorMaxHP`
The parsed version is: `[timestamp] <type>:<TargetID>: <target> (loses|gains) the effect of <skillname> from <actor> ( for <duration> seconds)?\.( <stacks>)?`

Formats for `NetworkEffectResult`:
raw: `targetId|targetName|sequence|currentHp|maxHp|currentMp|maxMp|currentTp|maxTp|posX|posY|posZ|heading|<the remaining packet data as uint fields in groups of 4, groups which only consist of 0s are skipped>`
parsed: The same as above, just `:` instead of `|`.

Formats for `NetworkStatusList`:
raw: `targetId|targetName|JobLevels|currentHp|maxHp|currentMp|maxMp|currentTp|maxTp|posX|posY|posZ|heading|<list of statuses, each status is a uint as hex>`
parsed: The same as above, just `:` instead of `|`.

Formats for `NetworkUpdateHp`:
raw: `targetId|targetName|currentHp|maxHp|currentMp|maxMp|currentTp|maxTp|posX|posY|posZ|heading`
parsed: The same as above, just `:` instead of `|`.

You can look this up in the `FFXIV_ACT_Plugin.LogFile.dll`. It has all of the type IDs and log line formats. The raw log line (which shows up in the network log) is converted into the parsed version in `FFXIV_ACT_Plugin.Parse.dll`.

If you need anything else from those files, let me know.

**Comment by JLGarber** [comment]
Thanks so much for extracting that. That's going to save me some time figuring it out manually. I have no experience disassembling dlls, and the very brief look through the plugin file I took just now is way more than I can learn to navigate over lunch. But with the data you posted already I can just do the guide updates.

**Comment by quisquous** [comment]
This should be fixed now.
